climpred/bootstrap.py

```diff
@@ -497,16 +590,66 @@ def bootstrap_compute(
     # get comparison function
     comparison = get_comparison_class(comparison, ALL_COMPARISONS)
 
-    # Perfect Model requires `same_inits` setup.
-    isHindcast = True if comparison in HINDCAST_COMPARISONS else False
+    # Perfect Model requires `same_inits` setup
+    isHindcast = True if comparison.name in HINDCAST_COMPARISONS else False
     reference_alignment = alignment if isHindcast else 'same_inits'
 
-    for i in range(iterations):
-        # resample with replacement
-        smp_hind = _resample(hind, resample_dim)
-        # compute init skill
-        init_skill = compute(
-            smp_hind,
+    if hist is None:  # PM path, use verif = control
+        hist = verif
+
+    # slower path for hindcast and resample_dim init
+    if resample_dim == 'init' and isHindcast:
+        warnings.warn(f'resample_dim=`init` will be slower than resample_dim=`member`.')
+        (
+            bootstrapped_init_skill,
+            bootstrapped_uninit_skill,
+            bootstrapped_pers_skill,
+            pers_output,
+        ) = _bootstrap_hindcast_over_init_dim(
```

Comment:
Is this supposed to be a tuple? `(bootstrapped_init_skill, bootstrapped_uninit_skill, bootstrapped_pers_skill, pers_output) = ...` or without the parentheses `( )`? Maybe this just comes from `black`.

Reply:
just comes from `black`